app/controllers/application_controller.rb

```diff
@@ -18,12 +18,18 @@ class ApplicationController < ActionController::Base
       if exception.is_a?(RuntimeError)
         raise exception
       else
+        Raven.capture_exception(exception)
         redirect_to error_500_url
       end
     end
   end
 
 
+  def dummy_exception
+    Raise ArgumentError.new('Boom!')
```

Comment:
Consider adding a more descriptive message (so people looking at Sentry know where it came from).